docs/jitsi.md

```diff
@@ -0,0 +1,47 @@
+# Jitsi in Riot
+
+Riot uses [Jitsi](https://jitsi.org/) for conference calls, which provides options for
+self-hosting your own server and supports most major platforms.
+
+1:1 calls, or calls between you and one other person, do not use Jitsi. Instead, those
+calls work through a TURN server configured on your respective homeservers.
+
+There's a number of ways to start a Jitsi call: the easiest way is to click on the 
+voice or video buttons near the message composer in a room with more than 2 people. This
+will add a Jitsi widget which allows anyone in the room to join.
+
+Integration managers (available through the 4 squares in the top right of the room) may
+provide their own approaches for adding Jitsi widgets, though these are now considered
+legacy and should only be used in specific circumstances.
```

Comment:
it's quite possible that you might want to use a specific jitsi server in general (not the one riot defaults to) in order to get jibri or other functionality. so i wouldn't declare it legacy per se.

Reply:
None of the integration managers offer the ability for the end user to change this. At the point where you're admining your own integration manager, you're already havign to change your Riot config...